src/schemes/boundary/open_boundary/system.jl

```diff
@@ -165,8 +165,6 @@ end
 
 update_callback_used!(system::OpenBoundarySPHSystem) = system.update_callback_used[] = true
 
-@inline source_terms(system::OpenBoundarySPHSystem) = nothing
-
```

Comment:
Why is this not needed anymore?

Reply:
It was never needed. Dispatch handles this case